packages/backend/package.json

```diff
@@ -7,9 +7,6 @@
   "backstage": {
     "role": "backend"
   },
-  "engines": {
```

Comment:
this is redundant and out of date -- you only need to specify this in the monorepo root package.json